SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+All versions of Openshot are security supported. This means we keep all versions of Openshot update to date with the latest security features around.
```

Comment:
Just out of curiosity — devil's advocate, here — what is that claim based on? What facts provide the basis for it? As we say on Wikipedia, `[citation needed]`.